app/scripts/controllers/registration.js

```diff
@@ -375,9 +375,16 @@ angular
             var savedAnswer = _.find(currentRegistrantOriginalAnswers, {
               id: a.id,
             });
+            let invalidBlocks = validateRegistrant.validate(
+              conference,
+              _.find(currentRegistration.registrants, {
+                id: currentRegistrant.id,
+              }),
+            );
             if (
-              angular.isUndefined(savedAnswer) ||
-              !angular.equals(savedAnswer.value, a.value)
+              (angular.isUndefined(savedAnswer) ||
+                !angular.equals(savedAnswer.value, a.value)) &&
+              !invalidBlocks.includes(a.blockId)
```

Comment:
Is there a chance `invalidBlocks` could come back null or undefined?

Reply:
Hmm I don't think so. But I will check to make sure it finds a registrant before assigning it.